src/Windows/Avalonia.Win32/WindowImpl.cs

```diff
@@ -37,10 +37,21 @@ public partial class WindowImpl : IWindowImpl, EglGlPlatformSurface.IEglWindowGl
                 { WindowEdge.West, HitTestValues.HTLEFT }
             };
 
+        private struct SavedWindowInfo
+        {
+            public WindowStyles Style { get; set; }
+            public WindowStyles ExStyle { get; set; }
+            public RECT WindowRect { get; set; }
+        };
+
+        private SavedWindowInfo _savedWindowInfo;
+        private bool _fullScreen;
```

Comment:
This probably should be in `WindowProperties`.

Reply:
done.